ui.content/pom.xml

```diff
@@ -57,6 +57,18 @@
                             <destinationFile>${basedir}/src/main/content/jcr_root/conf/venia/settings/cloudconfigs/commerce/.content.xml</destinationFile>
                         </configuration>
                     </execution>
+                    <execution>
```

Comment:
There is an easier way to do that, simply add `/var/commerce/products/venia-default` like you did and exclude it from the `cloud` profile like I did for https://github.com/adobe/aem-cif-guides-venia/blob/main/ui.apps/pom.xml#L141-L145

You'd have to test, but I think here you can simply add `<exclude>/var</exclude>`

The "copy" trick was only done because in this particular case, we have two different versions of the same content node for the two profiles.